eclass-writing/text.xml

```diff
@@ -53,12 +53,13 @@ does not require a new eclass will be suggested.
 </p>
 
 <p>
-Before updating <c>eutils</c> or a similar widely used eclass, it is best to email
-the gentoo-dev list. It may be that your proposed change is broken in a way you
-had not anticipated, or that there is an existing function which performs the
-same purpose, or that your function may be better off in its own eclass. If you
-don't email gentoo-dev first, and end up breaking something, expect to be in a
-lot of trouble.
+Before updating any eclass that is used by a more than few packages (as a rule
```

Comment:
I think you meant "more than a few" here.

Reply:
Fixed, thanks.